src/Event/Event.js

```diff
@@ -51,8 +50,8 @@ const eventPropType = PropTypes.shape({
   color: PropTypes.string,
   id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
   description: PropTypes.string,
-  startDate: PropTypes.instanceOf(moment).isRequired,
-  endDate: PropTypes.instanceOf(moment).isRequired,
+  startDate: PropTypes.object.isRequired,
+  endDate: PropTypes.object.isRequired,
```

Comment:
What's motivating the move to Object instead?

Reply:
I thought these props were `moment` variables, but apparently not as they were raising warnings. This change causes the warning to not show up.